### DISCUSSION TITLE
List of Figures numbering by chapter

### DISCUSSION THREAD

**Discussion by poliveiraIPB** [joaomlourenco/novathesis#348] [Q&A] [answered]
Hi, my list of figures is generated, separated by chapter, but i don't know, how can i numbering it in this way.
To appear like:

1.1
1.2

2.1
2.2

Also the same occur with tables.

<img width="935" alt="Captura de ecrã 2022-08-17, às 11 34 03" src="https://user-images.githubusercontent.com/14893611/185098969-56a28443-dea4-4bcf-acc0-a464857d087d.png">

**Comment by poliveiraIPB** [comment]
@joaomlourenco can you help me with this ?

**Reply by joaomlourenco** [reply]
@poliveiraIPB 
You're from Universidade do Minho, right?
You can but you shouldn't.  The UMinho rules define the numbering as it is.  :)

**Reply by poliveiraIPB** [reply]
Yes.

Ok, thank you.

**Reply by b-pereira** [reply, marked as answer]
Just to add something (if someone has the same issue again and comes across this thread), always **read the documentation**. In this case, [UMINHO Rectoral Order RT-31/2019](https://alunos.uminho.pt/PT/estudantes/Formataes/1_Despacho_RT-31_2019.pdf) where the formatting guidelines are. While this template helps to automate a lot of stuff, it doesn't replace the formatting guidelines.

> 17. Tables, charts, graphs, figures, etc. must be numbered, in each case, from 1 to N, and contain a synthetic title, which translates the corresponding title.

Don't ask! RTFM! 😀

Also, I noted that there are citations in the list of figures. From what I've seen, it's a common error. Use this:

```latex
\caption[''short'']{''long''}
```
The "long" is what appears in the caption. The "short" is what appears in the list of figures. Put the citation only in "long".

Taken from [here](https://en.m.wikibooks.org/wiki/LaTeX/Floats,_Figures_and_Captions#Lists_of_figures_and_tables)
